README.rst

```diff
@@ -23,6 +23,17 @@ Please don't commit changes with reStructuredText syntax errors that cause PEP
 generation to fail, or result in major rendering defects relative to what you
 intend.
 
+ReadTheDocs preview
+-------------------
+
+For every PR, we automatically create a preview of the rendered PEPs using
+ReadTheDocs. You can find it in the branch action box at the bottom of the
+PR page:
+
+- Click "Show all checks" to expand the box
+- Find the line for ``docs/readthedocs.org:pep-previews``
+- Click on "Details" to the right
```

Comment:
```suggestion
1. Click "Show all checks" to expand the checks section
2. Find the line for ``docs/readthedocs.org:pep-previews``
3. Click on "Details" to the right
```

* This list has order, so it probably should be an ordered list
* "box" -> "checks section" to be more specific about what it expands, since we just called the whole thing a "box"

Reply:
Also, I'd normally use `:guilabel:` here to get appropriate styling on the UI text, but I'm not sure how GitHub's reST renderer will handle it, and its not _that_ important.